plone/app/vocabularies/search.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from Acquisition import aq_get
+from zope.i18n import translate
+from zope.interface import implementer
+from zope.schema.interfaces import IVocabularyFactory
+from zope.schema.vocabulary import SimpleTerm
+from zope.schema.vocabulary import SimpleVocabulary
+from zope.site.hooks import getSite
+
+
+@implementer(IVocabularyFactory)
+class SearchIndexesVocabulary(object):
+    """Vocabulary factory for really user friendly portal types.
+
+      >>> from zope.component import queryUtility
+      >>> from plone.app.vocabularies.tests.base import create_context
+
+      >>> name = 'plone.app.vocabularies.SearchIndexes'
+      >>> util = queryUtility(IVocabularyFactory, name)
+      >>> context = create_context()
+
+      >>> indexes = util(context)
+      >>> indexes
+      <zope.schema.vocabulary.SimpleVocabulary object at ...>
+
+      >>> len(indexes.by_token)
+      3
+
+      >>> term = indexes.by_token['Date']
+      >>> term.title, term.token, term.value
+      (u'date (newest first)', 'Date', 'Date')
+    """
+
+    def __call__(self, context):
+        site = getSite()
+        request = aq_get(site, 'REQUEST', None)
+
+        items = []
+        items.append((translate(u'relevance', context=request), 'relevance'))
+        items.append((translate(u'date (newest first)', context=request), 'Date'))
+        items.append((translate(u'alphabetically', context=request), 'sortable_title'))
```

Comment:
But `i18ndude` won't pick these strings up, because it does not recognise them. So unless these strings are already picked up from a different place, these won't be in any `.po` file.